cumulus_library/databases/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .athena import AthenaDatabaseBackend, AthenaParser
+from .base import DatabaseBackend, DatabaseCursor, DatabaseParser
+from .duckdb import DuckDatabaseBackend, DuckDbParser
+from .utils import create_db_backend, read_ndjson_dir
+
+__all__ = [
```

Comment:
I would recommend not including __all__, especially in a case like this where you don't have other code - it's really only useful if you want to restrict the symbols being exported. But here, all the code is elsewhere and you're just allow-listing-by-importing already.

Reply:
as discussed off thread, added an ignore for this.